tests/stress/test_stress_routes.py

```diff
@@ -26,13 +26,13 @@ def announce_withdraw_routes(duthost, localhost, ptf_ip, topo_name):
 
     logger.info("ipv4 route used {}".format(get_crm_resources(duthost, "ipv4_route", "used")))
     logger.info("ipv6 route used {}".format(get_crm_resources(duthost, "ipv6_route", "used")))
-    sleep_to_wait(CRM_POLLING_INTERVAL * 5)
+    sleep_to_wait(CRM_POLLING_INTERVAL * 20)
 
     logger.info("withdraw ipv4 and ipv6 routes")
     localhost.announce_routes(topo_name=topo_name, ptf_ip=ptf_ip, action="withdraw", path="../ansible/")
 
     wait_until(MAX_WAIT_TIME, CRM_POLLING_INTERVAL, 0, lambda: check_queue_status(duthost, "inq") is True)
-    sleep_to_wait(CRM_POLLING_INTERVAL * 5)
+    sleep_to_wait(CRM_POLLING_INTERVAL * 20)
```

Comment:
* Could it be a SONiC image regression?
* Is there a way to use something like wait_until instead of simple sleep?